README.MD

```diff
@@ -389,11 +389,45 @@ sudo chown <b>user</b>:<b>group</b> /etc/daft/lockfiles
 ## 2.5 Setting BeagleBone Black to boot from NFS
 
 Setting BBB to boot from NFS is done by changing `/uEnv.txt` file contents on a
-BBB image (this means the uEnv.txt file in the BBB itself and not in the NFS files) to this:
+BBB image (this means the uEnv.txt file in the BBB itself and not in the NFS files).
+
+To change the `/uEnv.txt` file, boot the BBB manually. Follow the section [Beaglebone Black booting](#61-beaglebone-black-booting) on how to do it.
+When the BBB has succesfully booted (and you are connected to it via screen or ssh) do the following command:
+```
+lsblk
+```
+This should bring up a list of disks and partitions available.
+```
+NAME         MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+mmcblk1boot0 179:16   0    2M  1 disk
+mmcblk1boot1 179:24   0    2M  1 disk
+mmcblk0      179:0    0 14.4G  0 disk
+└─mmcblk0p1  179:1    0   50M  0 part
+mmcblk1      179:8    0  3.6G  0 disk
+```
+Search for the correct partition (in this case it is `mmcblk0p1`). The correct partition is the one where your BBB booted
+from (internal memory, SD card or other media), here we are using a USB stick. Now mount the partition to /mnt folder in root. There should be a `mnt` folder
+(if not create a <b>mnt</b> folder in your pc_hosts `/daft/bbb_fs/`).
```

Comment:
If we have booted from internal memory, sd card or USB stick, creating `mnt` folder on PC hosts nfs won't really do anything. Also this `Follow the section [Beaglebone Black booting](#61-beaglebone-black-booting) on how to do it.` is for booting from NFS. So this guide mixes a bit booting from nfs and real hardware.